types/maxmsp/index.d.ts

```diff
@@ -1219,6 +1223,69 @@ declare class Maxobj {
      * others.
      */
     understands(message: string): boolean;
+
+	/**
+	 * Returns an Array value containing the names of available attributes for the object.
+	 */
+	getattrnames(): string[];
+
+	/**
+	 * Returns the value of the attribute specified by attribute_name. Lists are returned as JS Array objects.
+	 */
+    getattr(attribute_name: "assistshowspatchername" | "bottomtoolbarpinned" | "boxanimatetime" | "enablehscroll"
+        | "enablevscroll" | "fontface" | "fontsize" | "gridonopen" | "gridsnaponopen" | "helpsidebarclosed" | "isolateaudio"
+        | "lefttoolbarpinned" | "objectsnaponopen" | "openinpresentation" | "righttoolbarpinned" | "showontab" | "tallnewobj"
+        | "textjustification" | "toolbarvisible" | "toptoolbarpinned" | "workspacedisabled"): number;
+    getattr(attribute_name: "defaultfocusbox" | "description" | "digest" | "filepath" | "fontname"
+        | "globalpatchername" | "style" | "subpatcher_template" | "tags" | "title"): string;
+    getattr(attribute_name: "accentcolor" | "bgcolor" | "clearcolor" | "color"
+        | "editing_bgcolor" | "elementcolor" | "locked_bgcolor" | "patchlinecolor"
+        | "selectioncolor" | "stripecolor" | "textcolor" | "textcolor_inverse"): Color;
+    getattr(attribute_name: "openrect"): Rect;
+    getattr(attribute_name: "bgfillcolor"): any;
```

Comment:
NOTE: "bgfillcolor" seems to be some sort of jsobject, not sure why and honestly 
for now I don't care, that's why you see `any`

Reply:
Pete (a colleague of mine) clarified me that this attr is for gradients and mixed colors, thus it is a nested dictionary.

Leaving it as any for the time being.

Reply:
I've been testing this for a while and I am not sure I am happy with this approach, there are way too many objects in max and each of them seems to have it's own set of attributes.

If we hardcode all of them here we will have a huge mantenance cost over here, thus I will proceed to delete this changes and take a simpler approach at the expense of the user having to do type narrowing.